pilot/lib/pilot/build_manager.rb

```diff
@@ -67,6 +67,7 @@ def distribute(options, build = nil)
 
       unless config[:update_build_info_on_upload]
         if should_update_build_information(options)
+          UI.user_error!("Beta App Description must be at least 10 characters long.") if options[:beta_app_description].to_s.length < 10 && options[:beta_app_description].to_s.length > 0
```

Comment:
There are other lines called `update_build_information!` method. If it fails in all cases, I think we should validate with `verify_block` in `options.rb`.